test/src/unit/repo/core/model/bson/ut_repo_bson_factory.cpp

```diff
@@ -284,31 +284,34 @@ TEST(RepoBSONFactoryTest, MakeMaterialNodeTest)
 
 TEST(RepoBSONFactoryTest, MakeMetaDataNodeTest)
 {
-	std::vector<std::string> keys({ "one", "two", "three", "four", "five" }), values({ "!", "!!", "!!!", "!!!!", "!!!!!" });
-
 	std::string name = "MetaTest";
-
-	MetadataNode metaNode = RepoBSONFactory::makeMetaDataNode(keys, values, name);
-
+	std::unordered_map<std::string, repo::lib::RepoVariant> metaDataUnMap;
+
+	metaDataUnMap["one"]=std::string("!");
```

Comment:
can we test some non string values please.

Reply:
Now we have changed this to integer and are testing this.